R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -1,8 +1,51 @@
 #' @title Deephaven TableHandles
+#' @md
 #' @description
 #' A TableHandle holds a reference to a Deephaven Table on the server, and provides methods for operating on that table.
 #' Note that TableHandles should not be instantiated directly by user code, but rather by server calls accessible from
-#' the `Client` class. See `?Client` for more information.
+#' the [`Client`][Client] class. See `?Client` for more information.
+#'
+#' @section
+#' Naming tables on the server:
+#' When a TableHandle is created, it is not automatically bound to a variable name on the server. This means that the
+#' TableHandle that gets created is _the only_ reference to the table that's been created. Importantly, the variable
+#' name given to the TableHandle is purely a _local_ variable, and has no relationship to that table's name on the server.
+#' For this reason, code like the following:
+#' ```r
+#' client <- Client$new(...)
+#' df1 <- data.frame(x = 1:10, y = 11:20)
+#' t1 <- client$import_table(df1)
+#' client$run_script("t2 = t1.update('z = x + y')")
+#' ```
+#' will not run, because the table referenced by the local variable `t1` is not actually named `t1` on the server. To
```

Comment:
Saying "not actually named `t1` on the server" feels like an implied "but named something else".   I think the point is that is not named at all, so I suggest replacing:

"not actually named `t1` on the server" => "not actually named on the server"